addons/account/static/src/components/tax_totals/tax_totals.xml

```diff
@@ -39,7 +39,7 @@
     </t>
 
     <t t-name="account.TaxTotalsField">
-        <table t-if="totals">
+        <table t-if="totals" class="float-end">
```

Comment:
Why you need to do change here? :thinking: 
(Do we want to change tax total alignment from all that place :thinking: )

Reply:
Yes, the same issue is occurring in multiple modules. Therefore, applying this change will correct the tax total alignment across all affected modules, ensuring a consistent user experience.